### PR TITLE
fix(github): add clone url to repo and concat name and owner

### PR DESCRIPTION
**Pull request by warren830**
### Summary
In this pr, we did the following:
1. add cloneUrl to githubRepo for convenience of bp v200
2. concat owner and repo for _github_tool_repos
3. when making plan for bp v200, we dont need to request api any more. Because backend can get the info from `PUT SCOPE`
4. combine owner and name in githubApiParams

### Does this close any open issues?
Relate to #3803 

### Screenshots
image

Here is the result of triggering advanced mode
image
Here is the result of triggering bp v200
image

Here is  the result we posted bpV200, and we got cloneUrl from db:
image

### Other Information
Any other information that is important to this PR.

### CODE REVIEW

File: plugins/github/models/repo.go [hunk 1]
```diff
@@ -34,9 +34,10 @@ type GithubRepo struct {
 	Language             string     `json:"language" gorm:"type:varchar(255)" mapstructure:"language,omitempty"`
 	ParentGithubId       int        `json:"parentId" mapstructure:"parentGithubId,omitempty"`
 	ParentHTMLUrl        string     `json:"parentHtmlUrl" mapstructure:"parentHtmlUrl,omitempty"`
+	CloneUrl             string     `json:"cloneUrl" gorm:"type:varchar(255)" mapstructure:"cloneUrl,omitempty"`
 	CreatedDate          *time.Time `json:"createdDate" mapstructure:"-"`
 	UpdatedDate          *time.Time `json:"updatedDate" mapstructure:"-"`
-	common.NoPKModel     `json:"-" mapstructure:"-"`
+	common.NoPKModel     `json:"-" mapstructure:"-" swaggerignore:"true"`
```
Comment (klesh) [inline plugins/github/models/repo.go:40]: Will these make the `createdAt` `updatedAt` disappear from response body?
if so, we should move it to :
image

**Review by klesh — COMMENTED** [review]
Please update the PrepareTaskData function as well.

File: plugins/github/models/migrationscripts/20221214_concat_owner_name_for_repo.go [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+		[]string{"name", "owner_login"},
+		func(src *githubRepoToBeTransform) (*githubRepoToBeTransform, errors.Error) {
+			if src.OwnerLogin != "" {
+				src.Name = fmt.Sprintf("%s/%s", src.OwnerLogin, src.Name)
+			}
+			return &githubRepoToBeTransform{
+				GithubId:     src.GithubId,
+				ConnectionId: src.ConnectionId,
+				Name:         src.Name,
+				OwnerLogin:   "",
```
Comment (klesh) [inline plugins/github/models/migrationscripts/20221214_concat_owner_name_for_repo.go:50]: So, why do we keep this empty field?

Comment (warren830) [reply plugins/github/models/migrationscripts/20221214_concat_owner_name_for_repo.go:50]: fixed

Comment (warren830) [reply plugins/github/models/migrationscripts/20221214_concat_owner_name_for_repo.go:50]: fixed

**Review by mappjzc — DISMISSED** [review]

**Review by mappjzc — APPROVED** [review]

**Pull request merged**